### ISSUE TITLE
How to force update single stream with fix-notation?

### ISSUE THREAD

**Issue by OlafHaag** [majkinetor/au#211]
Hey!
I have [this package](https://github.com/OlafHaag/chocolatey-packages/tree/master/automatic/jamovi.install) and it has 2 streams, a stable release and a current pre-release stream with latest features. In the last automatic update on appveyor the current stream was updated, but it seems the same version of the software was updated after that by the vendor, because the hashes don't match anymore.

I couldn't force push the same version again to choco community stream, because the pre-releases are automatically approved, even if automatic testing fails. So I wanted to push the package with fix-notation, in which I failed.
[Here's the update script](https://github.com/OlafHaag/chocolatey-packages/blob/master/automatic/jamovi/update.ps1).

So commit messages I tried were:
`[AU jamovi.install jamovi.portable]`  Resulted in trying to push the already approved stable release.
`[AU jamovi.install\current jamovi.portable\current]` Couldn't push, because package version _1.2.22.0-current_ already approved.
`[AU jamovi.install:1.2.22.0\current jamovi.portable:1.2.22.0\current]` Invalid Version
`[AU jamovi.install:1.2.22.0-current jamovi.portable:1.2.22.0-current]` Package version already exists.
`[AU jamovi.install\current:1.2.22.0 jamovi.portable\current:1.2.22.0]` Now this last one posed a problem, as it did update the current stream to version 1.2.22.0 instead of 1.2.22.0-current. It did update stream _current_, but did not add fix-notation and _-current_.
What I want is a _jamovi.install.1.2.22.20200530-current_ version and I can't figure out how to do it. Does the update script need to change or my commit messages?

I'd appreciate any insights.

**Comment by AdmiringWorm** [comment]
The format for updating streams is:
```
[AU package\stream:version]
```

However, whenever you specify the version you are telling AU to use that specific version for that stream.
AU also won't use a fix notation when all four parts of the package version are used, so you would have to specify the version you want yourself in that case.

For instance:
```
[AU jamovi.install\current:1.2.22.20200530-current]
```

I would not recommend using that as a version number though, for instance, what would you do if the software devs released the version `1.2.22.1-current`? That version would then be lower than the version you created as a fix version.
I would instead recommend the following for 4-part pre-releases:
```
[AU jamovi.install\current:1.2.22.0-current-20200530]
```

**Comment by OlafHaag** [comment]
Thanks for the info!
